### PR TITLE
Added credentials section to on-gke setup instructions.

### PR DESCRIPTION
**Pull request by kevensen**
Thanks for opening a Pull Request!

Here's a handy checklist to ensure your PR goes smoothly.

- [x] I signed Google's [Contributor License Agreement](https://opensource.google.com/docs/cla/)

### CODE REVIEW

File: _docs/_latest/setup/forseti-on-gke.md [hunk 1]
```diff
@@ -15,8 +15,37 @@ The following tools are required:
...
+setup.sh -o ORG_ID -p PROJECT_ID -k
+```
+The above command will result in a *credentials.json* being downloaded to the your current working directory.
+
+### Export the Credentials
+Set the appropriate environment variable to the path of the service account JSON
+key file:
+
+```sh
+export GOOGLE_APPLICATION_CREDENTIALS=credentials.json
```
Comment (gkowalski-google) [inline _docs/_latest/setup/forseti-on-gke.md]: FYI - The [setup script](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/helpers/setup.sh#L121) will now export this environment variable. In order for that export to stick around after the script is executed, you need to source it.

File: _docs/_latest/setup/forseti-on-gke.md [hunk 2]
```diff
@@ -15,8 +15,37 @@ The following tools are required:
...
 
+## Service Account Credentials
+Terraform uses an IAM Service Account to deploy and configure resources on behalf of the user.  This
+service account and key can be created by executing the [setup.sh](https://github.com/forseti-security/terraform-google-forseti/blob/master/helpers/setup.sh)
+helper script.
+
+### Obtain the Setup Script
+The following will download the setup script locally.
+```sh
+curl --location --remote-name https://raw.githubusercontent.com/forseti-security/terraform-google-forseti/master/helpers/setup.sh
```
Comment (gkowalski-google) [inline _docs/_latest/setup/forseti-on-gke.md]: Food for thought. We have these same instructions on the TF repo and soon to be on the install page of the website. Do we need to duplicate these steps here?

File: _docs/_latest/setup/forseti-on-gke.md (hunk 1 above)
Comment (kevensen) [reply _docs/_latest/setup/forseti-on-gke.md]: Ok.  I can take this out.

File: _docs/_latest/setup/forseti-on-gke.md (hunk 2 above)
Comment (kevensen) [reply _docs/_latest/setup/forseti-on-gke.md]: @gkowalski-google Will the instructions in the the install page of the website be phrased in such a way that they these steps are for both GKE and GCE.  Historically, the install page has just been for GCE.  To be clear, I have no issue having "common" instructions in a "common" place.

Comment (kevensen) [reply _docs/_latest/setup/forseti-on-gke.md]: Regarding cloning the repo, I'll make it more generic.  How about:
```
Obtain the [setup.sh](https://github.com/forseti-security/terraform-google-forseti/blob/module-release-5.0.0/helpers/setup.sh) helper script from the terraform-google-forseti repo.
```

Comment (gkowalski-google) [reply _docs/_latest/setup/forseti-on-gke.md]: I think the changes @hshin-g can be updated to support both. Perhaps we can instruct users to run through the setup script, and then either continue reading on to deploy GCE or _click here_ to deploy GKE.

**Comment by blueandgold** [comment]
Please review Gregg's comment before merge.

**Pull request merged**